admin/runner.py

```diff
@@ -27,21 +48,21 @@ class CommandProtocol(LineOnlyReceiver, object):
         If the command finished successfully, will fire with ``None``.
         Otherwise, errbacks with the reason.
     :ivar file-like output: For logging.
+
+    :ivar defaultdict _fds: Mapping from fds to `_LineParsers`.
```

Comment:
:question: It seems to be more standard to write "file descriptors" - at least it'd be easier for me to read.